server/user_account/views.py

```diff
@@ -44,9 +44,10 @@ def get_permissions(self):
         :param: actions
         :return: permission
         """
-        if self.action in ['retrieve', 'update', 'partial_update']:
-            permission_classes = [IsAuthenticated, IsCurrentUserTargetUser | IsSystemAdmin]
-
+        if self.action in ['retrieve']:
+            permission_classes = [IsAuthenticated, (IsSystemAdmin | IsInventoryManager)]
+        elif self.action in ['update', 'partial_update']:
+            permission_classes = [IsAuthenticated, IsCurrentUserTargetUser]
         # TODO: Validate requested user id matches requested organization in DB
```

Comment:
To elaborate on my previous comment, maybe we can add another permission to check if they are only touching the fields you want the user to touch?

But again, I'm not sure this fits into what we discussed, and this might be extra work for us.